components/src/Checkbox/Checkbox.js

```diff
@@ -71,7 +70,7 @@ const BaseCheckbox = props => {
       <CheckboxWrapper disabled={ disabled }>
         <CheckboxInput type="checkbox" { ...props } />
         <VisualCheckbox disabled={ disabled } checked={ checked } />
-        {labelText}
+        <Text disabled={ disabled } mb={ 0 }> {labelText} </Text>
```

Comment:
Why do we need `mb={ 0 }`? I see it throughout this review.
I wonder if the default should be 0. To follow the "no margin/padding on atoms" guideline.

Reply:
Opacity looks good for now. Maybe it will be slightly adjusted when we implant this somewhere.

Reply:
Our current Text component uses a margin bottom for spacing. The intended fix is in this ticket: https://nulogy-go.atlassian.net/browse/NDS-539 Where we discussed that Text should not have mb, and that there should be a [Paragraph?] component that is essentially Text with a default mb. That way we can remove all these instances of mb={ 0 }. For the sake of this PR we can ignore it and that work should be handled in that ticket